solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -84,44 +74,19 @@ for ip in $PUBLIC_IP;do rsync -avzrP -e 'sudo ssh -i cortx.pem -o StrictHostKeyC
 
 **Note:** Become the **root** user after logged in to the primary node by running `sudo su` command.
```

Comment:
We can remove this as we are using `ssh` command from local system to run the script

Reply:
done